easybuild/easyblocks/m/mcr.py

```diff
@@ -137,6 +131,17 @@ def make_module_extra(self):
         """Extend PATH and set proper _JAVA_OPTIONS (e.g., -Xmx)."""
         txt = super(EB_MCR, self).make_module_extra()
 
+        # determine subdirectory (e.g. v84 (2014a, 2014b), v85 (2015a), ...)
+        subdirs = os.listdir(self.installdir)
+        # skip the easybuild folder if it exists
+        if "easybuild" in subdirs:
+            i = subdirs.index("easybuild")
```

Comment:
This shouldn't be hardcoded, since this can be configured (via `--logfile-format`).

Either determine the directory name to filter out via `log_path`, or use a regex to grab the directory named `v[0-9]*`?

With `log_path`:

```python
from easybuild.tools.config import log_path
...
    eb_logdir_name = os.path.basename(log_path())
    if eb_logdir_name in subdirs:
        subdirs.remove(eb_logdir_name)
```